pkg/ruler/manager.go

```diff
@@ -49,7 +49,7 @@ type DefaultMultiTenantManager struct {
 
 	// Prometheus rules managers metrics.
 	userManagerMetrics *ManagerMetrics
-
+	notifierMetrics    *NotifierMetrics
```

Comment:
nit to put this next to the map of notifiers instead of next to the ruler managers